src/main/webapp/auth.html

```diff
@@ -0,0 +1,44 @@
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Authentication Page</title> 
+    <link rel="stylesheet" href="style.css">
+  </head>
+ <body onload="googleSignIn()">
+    
+
+<!-- Initialize firebase products -->
+ <script src="https://www.gstatic.com/firebasejs/7.15.5/firebase-app.js"></script>
+ <script defer src="https://www.gstatic.com/firebasejs/7.15.5/firebase-auth.js"></script>
+
+ <script>
+  // Your web app's Firebase configuration
+  var firebaseConfig = {
+    apiKey: "AIzaSyAGk_4TZiVf_TslWON5hNQDYVODuJifP7A",
+    authDomain: "com-sdk-dashboard-step-2020.firebaseapp.com",
+    databaseURL: "https://com-sdk-dashboard-step-2020.firebaseio.com",
+    projectId: "google.com:sdk-dashboard-step-2020",
+    storageBucket: "undefined",
+    messagingSenderId: "412575630040",
+    appId: "1:412575630040:web:824b2d31f4e1f06b100dad",
+    measurementId: "G-QLFTN6MHB0"
+    
+  };
+  // Initialize Firebase
+  firebase.initializeApp(firebaseConfig);
+
+  // Redirect to Google Sign In
+  function googleSignIn() {
+
+    var provider = new firebase.auth.GoogleAuthProvider();
+
+    firebase.auth().signInWithRedirect(provider).then(function(result) {
+        console.log(result, "Success Google Account Linked");
+    }).catch(function(error){
+        console.log(error, "failed to sign-in");
```

Comment:
nit: We generally try to keep log messages in same formats to keep them easier to read. You can change this to "Sign in failed" and the success message to "Sign in succeeded".